### PR TITLE
Update Travis CI and Development server redirection

### LINKED ISSUE #652: Update TravisCI badge and Development server link on README
## Description
As a contributor,
I need README updated with the project deployment status,
so that I can be up to date.

Some links are not up to date, which can lead to confusion, like the Travis CI badge for develop branch, and the Development server link.

### Update [Required]
- [ ] Change Travis CI develop badge link to https://travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop
- [ ] Change development server link to https://mentorship-backend-temp.herokuapp.com/ 

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
30 minutes

### PR DESCRIPTION
**Pull request by Neilblaze**
### Description

- [x]  Updated TravisCI badge (master+develop) 

- [x]  Updated development server link to `https://mentorship-backend-temp.herokuapp.com/`

Fixes #652 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Neilblaze** [comment]
@gaurivn can you please review #655 ?

**Comment by gaurivn** [comment]
Sure, @Neilblaze, please update the title of your PR to 
```
docs: Update Travis CI badge and Development server link on README
```

**Comment by Neilblaze** [comment]
@gaurivn done 🎉

**Comment by gaurivn** [comment]
> Sure, @Neilblaze, please update the title of your PR to
> 
> ```
> docs: Update Travis CI badge and Development server link on README
> ```

@Neilblaze, kindly change your PR title back to this one.
Kindly read this [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide)

**Comment by gaurivn** [comment]
Okay

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
...
-| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://travis-ci.org/systers/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](http://systers-mentorship-dev.eu-central-1.elasticbeanstalk.com/) |
+| [master](https://github.com/anitab-org/mentorship-backend/tree/master) | [![Build Status](https://api.travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=master)](https://www.codacy.com/app/systers/mentorship-backend) | [Production](http://systers-mentorship.eu-central-1.elasticbeanstalk.com/) |
```
Comment (gaurivn) [inline README.md:5]: The issue was only to change Travis CI develop badge link and development server link... no need to make changes to the master badge

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://api.travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](https://mentorship-backend-temp.herokuapp.com/) |
```
Comment (gaurivn) [inline README.md]:
```suggestion
| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](https://mentorship-backend-temp.herokuapp.com/) |
```

Comment (gaurivn) [reply README.md]: This url would suit better.

**Review by gaurivn — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Neilblaze) [reply README.md]: Ok! Working on it.

File: README.md (hunk 1 above)
Comment (Neilblaze) [reply README.md:5]: Sorry about that. Now it should be working!

Comment (Neilblaze) [reply README.md:5]: Thanks for the edit :)

**Comment by gaurivn** [comment]
I didn't get how you made the previous commit, it says I co-authored it, can you please remove it and make a new one.

**Comment by Neilblaze** [comment]
Extremely sorry, I am making a new one.

**Comment by gaurivn** [comment]
> Extremely sorry, I am making a new one.

Okay while creating commit please don't include links in the commit name like the "fixed #652" one, kindly follow the commit message style guide for the same.

**Pull request closed without merging**